src/parser/deathknight/unholy/modules/features/checklist/Component.tsx

```diff
@@ -3,14 +3,15 @@ import PropTypes from 'prop-types';
 
 import SPELLS from 'common/SPELLS';
 import SpellLink from 'common/SpellLink';
+
 import Checklist from 'parser/shared/modules/features/Checklist';
 import Rule from 'parser/shared/modules/features/Checklist/Rule';
 import Requirement from 'parser/shared/modules/features/Checklist/Requirement';
 import PreparationRule from 'parser/shared/modules/features/Checklist/PreparationRule';
 import GenericCastEfficiencyRequirement from 'parser/shared/modules/features/Checklist/GenericCastEfficiencyRequirement';
 
-const UnholyDeathKnightChecklist = ({ combatant, castEfficiency, thresholds }) => {
-  const AbilityRequirement = props => (
+const UnholyDeathKnightChecklist = ({ combatant, castEfficiency, thresholds }: any) => {
```

Comment:
Avoid explicit `any`; leave as JS here as it's the only change added (or type the props)

Reply:
Set it back to JS as typing the props triggered a cascade of needing to convert runetracker and its dependencies to TS